core/CMakeLists.txt

```diff
@@ -156,10 +156,10 @@ if(CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   )
 endif()
 
-set_target_properties(tangram-core
-  PROPERTIES
-  CXX_STANDARD 14
-  CXX_STANDARD_REQUIRED ON
+# C++14 is required for compiling tangram-core and for using the interface headers.
+# Making this option PUBLIC declares it as a requirement for targets linking against tangram-core.
+target_compile_options(tangram-core
+  PUBLIC -std=c++14
```

Comment:
Can you `cxx_std_14` here. https://cmake.org/cmake/help/v3.8/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html

Reply:
That feature option isn't available until CMake 3.8. Some platforms don't have 3.8 available by default (e.g. Ubuntu 16.04).